jmeter-java-dsl/src/main/java/us/abstracta/jmeter/javadsl/core/DslStatisticsSummaryData.java

```diff
@@ -0,0 +1,129 @@
+package us.abstracta.jmeter.javadsl.core;
+
+import org.apache.jmeter.report.processor.MeanAggregator;
+import org.apache.jmeter.report.processor.PercentileAggregator;
+import org.apache.jmeter.report.processor.StatisticsSummaryData;
+
+public class DslStatisticsSummaryData extends StatisticsSummaryData {
+
+	private final PercentileAggregator latencyPercentile1;
+	private final PercentileAggregator latencyPercentile2;
+	private final PercentileAggregator latencyPercentile3;
+	private final MeanAggregator latencyMean;
+	private final PercentileAggregator latencyMedian;
```

Comment:
I see you added median for the new metrics, but maybe to be consistent we should add also for elapsed time?